src/app/[locale]/(default)/storage/page.tsx

```diff
@@ -74,69 +72,63 @@ export default function StoragePage({
   ];
 
   const filters = [
-    'select.popularity',
-    'select.sortDescending',
-    'select.sortAscending',
-    'select.name',
-  ] as const;
+    { name: t('select.popularity'), urlName: 'popularity' },
```

Comment:
is there a better way to do this? Maybe having a key in next-intl that returns the translation based on the valuekey. Also the valuekey can be an index numbers instead of a word